src/criteria_assessment/site_identification.jl

```diff
@@ -0,0 +1,214 @@
+"""Methods for identifying potential deployment locations."""
+
+using FLoops, ThreadsX
+
+"""
+    proportion_suitable(subsection::BitMatrix, window::Tuple=(-4,5))::Matrix{Int16}
+
+Calculate the the proportion of the subsection that is suitable for deployments.
+Subsection is the surrounding a rough hectare area centred on each cell of a raster marked
+as being suitable according to user-selected criteria.
```

Comment:
The pixels on edges consider a smaller area rather than the calculated window being shifted inward. Should that be documented?